### ISSUE TITLE
Explorer server process is shutdown unexpectly without any message

### ISSUE THREAD

**Issue by HaoyangLiu** [forbole/big_dipper#198]
**Describe the bug**

git commit hash: 7c405810243f76773c9e49ff4c9d6cc3b536f7ce
I started to cosmos explorer in local mode
```
meteor npm install
meteor update
meteor --settings settings.json
```
At first, it works well. However, later, the process was shutdown unexpectly without any error message

**Comment by kwunyeung** [comment]
Difficult to address the issue. Can you elaborate more?

**Comment by kwunyeung** [comment]
@HaoyangLiu this looks irrelevant and hard to follow. If there is no any new input, I will close this issue.
